doc/api/esm.md

```diff
@@ -450,6 +454,8 @@ cjs === 'cjs'; // true
 
 ## Experimental JSON modules
```

Comment:
Should we perhaps remove "Experimental" from the title if the stability index is just below?

Reply:
Sure we can do that.

Reply:
+1, this would allow to keep the same heading when/if this goes out of experimental; every time we update a heading on our docs, we take the risk of breaking links and we should avoid that when possible.